lib/libs/email/content/withdrawPackage/emailTemplates/WaiverCMS.tsx

```diff
@@ -1,32 +1,28 @@
 import { CommonEmailVariables, Events } from "shared-types";
-import { SpamWarning, PackageDetails, BasicFooter } from "../../email-components";
+import { SpamWarning, PackageDetails, BasicFooter, Divider } from "../../email-components";
 import { BaseEmailTemplate } from "../../email-templates";
 
 export const WaiverCMSEmail = ({
   variables,
 }: {
   variables: Events["WithdrawPackage"] & CommonEmailVariables;
-}) => {
-  const previewText = `Withdrawal of ${variables.authority} ${variables.id}`;
-  const heading =
-    "The OneMAC Submission Portal received a request to withdraw the package below. The package will no longer be considered for CMS review:";
-  return (
-    <BaseEmailTemplate
-      previewText={previewText}
-      heading={heading}
-      applicationEndpointUrl={variables.applicationEndpointUrl}
-      footerContent={<BasicFooter />}
-    >
-      <PackageDetails
-        details={{
-          "State or territory": variables.territory,
-          Name: variables.submitterName,
-          "Email Address": variables.submitterEmail,
-          "Waiver Number": variables.id,
-          Summary: variables.additionalInformation,
-        }}
-      />
-      <SpamWarning />
-    </BaseEmailTemplate>
-  );
-};
+}) => (
+  <BaseEmailTemplate
+    previewText={`Waiver Package ${variables.id} Withdraw Request`}
+    heading="The OneMAC Submission Portal received a request to withdraw the package below. The package will no longer be considered for CMS review:"
+    applicationEndpointUrl={variables.applicationEndpointUrl}
+    footerContent={<BasicFooter />}
+  >
+    <Divider />
```

Comment:
Was the `Divider` a requested addition? I don't see it in the [template](https://confluenceent.cms.gov/display/MACPRO3/Email+Notifications+for+Package+Actions#EmailNotificationsforPackageActions-CMSUsers.20). Just curious

Reply:
I see that the State version below has the `<Divider />` below the `PackageDetails`, maybe this is meant to go there for consistency?

Reply:
Divider is a new addition, yes, but your other comment is valid. Resolving with https://github.com/Enterprise-CMCS/macpro-mako/pull/894/commits/23c2d64487f641c607dd54d282c3282e64811b99